docs/04-operation-guides/security/sec-06-access-expose-kiali-grafana.md

```diff
@@ -50,21 +50,27 @@ By default, Kyma does not expose Kiali, Grafana, and Jaeger. However, you can st
   </details>
 </div>
 
->**NOTE:** kubectl port-forward does not return. You will have to cancel it with Ctrl+C if you want to stop port forwarding.
+>**NOTE:** kubectl port-forward does not return. To stop port forwarding, cancel it with Ctrl+C.
```

Comment:
```suggestion
>**NOTE:** `kubectl port-forward` does not return. To stop port forwarding, cancel it with `Ctrl`+`C`.
```
I'd format it as code since this is a command (though not in its full form). Otherwise, I'd start the sentence with a capital letter.